### ISSUE TITLE
Create secret template for output command

### ISSUE THREAD

**Issue by troy0820** [troy0820/secretkube#8]
- Output command needs the template to pass in base64 encoded secret data
- Import `k8s.io/apimachinery/pkg/apis/meta/v1` as metav1 to handle namespace, etc data

**Note: Secret struct Data map[string][]byte**

 _The serialized form of the secret data is a base64 encoded string, representing the arbitrary (possibly non-string) data value here._

```
type Secret struct {
    metav1.TypeMeta
    metav1.ObjectMeta
    Data map[string][]byte
    Type SecretType
}
```

**Comment by troy0820** [comment]
StringData has been added to Secret.  Issue is blocked by #11

**Comment by troy0820** [comment]
Waiting on a unit test but validated through string conversion test in another program.

**Comment by troy0820** [comment]
Unit test landed in commit cd2552b7e5ca039ccd74bd8e6a7689bea7b4b37a
